glossary/en/terms/open-data/index.md

```diff
@@ -4,6 +4,6 @@ lang: en
 title: Open Data
 ---
 
-Data is open if it can be freely accessed, used, modified and shared by anyone for any purpose - subject only, at most, to requirements to provide attribution and/or [share-alike](/glossary/en/terms/share-alike-license). Specifically, open data is defined by the [Open Definition][od] and requires that the data be **A. Legally open**: that is, available under an open (data) license that permits anyone freely to access, reuse and redistribute **B. Technically open**: that is, that the data be available for no more than the cost of reproduction and in [machine-readable](../machine-readable/) and [bulk](../bulk/) form.
+Data is open if it can be freely accessed, used, modified and shared by anyone for any purpose - subject only, at most, to requirements to provide attribution and/or [share-alike](../share-alike-license/). Specifically, open data is defined by the [Open Definition][od] and requires that the data be **A. Legally open**: that is, available under an open (data) license that permits anyone freely to access, reuse and redistribute **B. Technically open**: that is, that the data be available for no more than the cost of reproduction and in [machine-readable](../machine-readable/) and [bulk](../bulk/) form.
```

Comment:
@jgkim - I also need to fix the [open definition link]... I'll do it after this pull request.